packages/commonwealth/client/scripts/views/components/header/login_selector.tsx

```diff
@@ -231,6 +249,8 @@ export const LoginSelectorMenuRight = ({
             $.get(`${app.serverUrl()}/logout`)
               .then(async () => {
                 await initAppState();
+                await resetWalletConnectSession();
```

Comment:
Are there any cases other than explicit logout where we might need to reset the session?

Reply:
On login failure probably